clientServiceClient/clientServiceClient.go

```diff
@@ -78,6 +78,17 @@ func (c *ClientServiceClient) GetPublicClient(ctx context.Context, clientID stri
 	return result, err
 }
 
+// InternalPatchBackup gets a client's public client information for clientID.
```

Comment:
Why is this namespaced as internal if this is public information?

Reply:
bad copy pasta comment 😬